docs/sdk_developers/checklist.md

```
@@ -54,6 +54,6 @@ The **Files Changed Tab** shows the exact **difference** between your branch and
 
 ## Need Help?
 
-- **Signing issues?** → [Signing Guide](signing.md)
-- **Merge conflicts?** → [Merge Conflicts Guide](merge_conflicts.md)
-- **Changelog format?** → [Changelog Guide](changelog_entry.md)
\ No newline at end of file
+- **Signing issues?** → [Signing Guide](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/signing.md)
+- **Merge conflicts?** → [Merge Conflicts Guide](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/merge_conflicts.md)
+- **Changelog format?** → [Changelog Guide](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/changelog-guide.md)
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)